sources/engine/Xenko.Engine/Rendering/Compositing/ForwardRenderer.cs

```diff
@@ -116,7 +116,7 @@ public partial class ForwardRenderer : SceneRendererBase, ISharedRenderer
         [DefaultValue(true)]
         public bool BindDepthAsResourceDuringTransparentRendering { get; set; } = true;
 
-        private Logger logger { get; } = GlobalLogger.GetLogger(nameof(ForwardRenderer));
+        private Logger Logger { get; } = GlobalLogger.GetLogger(nameof(ForwardRenderer));
```

Comment:
I was about to say that `private` fields are always lowercase. But then I realized it is actually a `private` property. 

I don't see the point of having a `private` property here. I think it should be converted to a `private readonly` field.